Gemfile

```diff
@@ -216,7 +216,7 @@ group :development, :test do
   gem 'rspec-instrumentation-matcher'
   gem 'rspec-its'
   gem 'rspec-rails'
-  gem 'rswag-specs'
+  gem 'rswag-specs', git: 'https://github.com/rswag/rswag', branch: 'master'
```

Comment:
Maybe this line is causing the issue? Since it's not a repo we own we might not want to point to `master` for security reasons. Is there a tag we can do instead of `master`?